rasa/cli/x.py

```diff
@@ -111,10 +111,24 @@ def _overwrite_endpoints_for_local_x(
     from rasa.utils.endpoints import EndpointConfig
     import questionary
 
+    # Checking if endpoint.yml has existing url and wait time values set, if so give warning we are overwriting
+    # the endpoint.yml file.
+    custom_wait_time_pulls = endpoints.model.kwargs.get("wait_time_between_pulls")
+    custom_url = endpoints.model.url
+
+
+    if custom_url is not None and custom_url != rasa_x_url:
```

Comment:
```suggestion
    if custom_url and custom_url != rasa_x_url:
```

Reply:
changed this as well forgot the old way was redundant